mmv1/third_party/terraform/tests/resource_dialogflowcx_intent_test.go.erb

```diff
@@ -0,0 +1,158 @@
+<% autogen_exception -%>
```

Comment:
If we don't use ruby tags in the file, we can remove the `.erb` suffix and remove the autogen exception.

Reply:
got it I removed tags and renamed file